src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -1,10 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_BuildTasksDirectory>$(MSBuildThisFileDirectory)</_BuildTasksDirectory>
+    <_BuildTasksDirectory Condition="Exists('$(RoslynTargetsPath)')">$(RoslynTargetsPath)\</_BuildTasksDirectory>
```

Comment:
what's the difference between this condition and something like `'$(RoslynTargetsPath)' != ''`?

Reply:
With `Exists` condition, we preserve the behavior that when `RoslynTargetsPath` doesn't exist, builds are still passing. I'm not sure if that's necessary, but we have pre-existing tests for it:

https://github.com/dotnet/roslyn/blob/b2c6d96250ac024efaa807e435a6bba3f610f6ee/src/Compilers/Core/MSBuildTaskTests/TestUtilities/DotNetSdkTestBase.cs#L234